apps/services/notifications/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,23 @@
-export const environment = {
+import assert from 'assert'
+
+const environment = {
```

Comment:
File replacement does not work anymore (with esbuildnx) to keep prod and dev environments separate.

Please consider using `@island.is/nest/config`, or join these files together.

Reply:
I see. I used the code generator to bootstrap the project, but it seems to be quite outdated :/